csrc/python_frontend/fusion_record.h

```diff
@@ -531,34 +531,32 @@ struct PadOpRecord : RecordFunctor {
   std::vector<int64_t> pad_widths_;
 };
 
-struct PermuteOpRecord : RecordFunctor {
-  PermuteOpRecord(
+template <serde::RecordType op_type>
+struct DimsOpRecord : RecordFunctor {
```

Comment:
renamed this RecordFunctor to be `DimsOpRecord` it now handles both `permute` and `stride_order`
cc'ing @zasdfgbnm